website/src/components/TeamProfileCard/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+export default function TeamProfile({name, children, githubUrl, twitterUrl}) {
+  return (
+    <div className="col col--6 margin-bottom--lg">
```

Comment:
I think the card shouldn't contain the col classes for better reusability. Col classes should remain in the parent class.

Reply:
thanks, totally missed that but totally agree :)

Good related blog post: https://visly.app/blog/layout-isolated-components